MdePkg/Include/IndustryStandard/IpmiNetFnGroupExtension.h

```diff
@@ -17,4 +20,67 @@
 // All Group Extension commands and their structure definitions to follow here
 //
 
+///
+/// Constants and structure definitions for "SBMR Appendix F Boot Progress Code
+/// reporting" command to follow here
+///
+
+//
+// Definitions for send progress code command
+//
+#define IPMI_GROUP_EXTENSION_BOOT_PROGRESS_CODE_SEND  0x02
+
+//
+// Definitions for get progress code command
+//
+#define IPMI_GROUP_EXTENSION_BOOT_PROGRESS_CODE_GET  0x03
+
+//
+// Definitions for send and get progress code command response
+//
+#define IPMI_GROUP_EXTENSION_BOOT_PROGRESS_CODE_COMPLETED_NORMALLY    0x00
+#define IPMI_GROUP_EXTENSION_BOOT_PROGRESS_CODE_COMPLETED_WITH_ERROR  0x80

```

Comment:
remove _WITH_ just _ERROR should be suffice.

Reply:
Will fix, thanks.